frontend/tests/controllers/QualityNominationTest.php

```diff
@@ -19,6 +19,20 @@ public function qualityNominationsDemotionStatusProvider(): array {
             ['warning', \OmegaUp\ProblemParams::VISIBILITY_PUBLIC_WARNING],
         ];
     }
+
+    /**
+     * A PHPUnit data provider for all the tests that can accept a column for search nominations.
+     *
+     * @return list<list<mixed>>
```

Comment:
así como en TypeScript hay que evitar el uso de `any` a toda costa, en PHP `mixed` es malo.

```suggestion
     * @return list<array{0: string, 1: int}>
```

Reply:
listo